python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/partition_scenarios.py

```diff
@@ -71,7 +72,13 @@
 ]
 
 
-time_multipartitioned_asset = [asset_def("asset1", partitions_def=time_multipartitions_def)]
+time_multipartitioned_asset = [
+    asset_def(
+        "asset1",
+        partitions_def=time_multipartitions_def,
+        auto_materialize_policy=AutoMaterializePolicy.eager(max_materializations_per_minute=2),
```

Comment:
similar to above, this MultiPartitionsDefinition has a static dimension of size 2, so setting the limit to 2 allows it to fill in one day's worth of data, aligning with our tests